sdk/src/client/node_api/indexer/query_parameters.rs

```diff
@@ -67,7 +67,7 @@ pub struct OutputQueryParameters {
     /// Filters outputs based on the presence of a specific native token.
     native_token: Option<TokenId>,
     /// Returns outputs that are unlockable by the bech32 address.
-    unlockable_by_address: Option<Bech32Address>,
+    unlockable_by_address: Option<String>,
```

Comment:
Should we have another type, Bech32AddressString, to at least validate it's correct Bech32?

Reply:
I think that's reasonable